nkms/crypto/utils.py

```diff
@@ -60,3 +126,11 @@ def __call__(self, splittable):
             messages.append(message)
             splittable = remainder
         return messages
+
+
+def fingerprint_from_key(public_key: Any):
+    """
+    Hashes a key using keccak-256 and returns the hexdigest in bytes.
+    :return: Hexdigest fingerprint of key (keccak-256) in bytes
+    """
+    return keccak_digest(bytes(public_key)).hex().encode()
```

Comment:
Why is Keccak here? Weren't we using Blake2b?

Reply:
If we need to store a fingerprint or something in a smart contract, it's easier for us to just use what Ethereum uses -- Keccak-256.

Reply:
Yeah, @tuxxy and I went around and around on this and decided that, since it's quite plausible that we'll need to do this in a smart contract, it makes sense to stick to keccak.

Reply:
Ok, great. Perhaps a comment?

Reply:
OK, good idea - coming up.

Reply:
✔️